README.md

```diff
@@ -25,3 +25,63 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+## Features and Benefits
+
+## Tooltip design
+
+Angular Tooltip is shipped with several built-in themes such as Material, Bootstrap, Fabric (Office 365), Tailwind CSS, and High Contrast. Users can customize any one of these built-in themes or create new themes to achieve their own desired look and feel. They can do this by either simply overriding SASS variables or using our Theme Studio application.
+
```

Comment:
remove this part, tooltip design(build in theme) is not a feature

Reply:
Removed